modules/ROOT/pages/enable-inbound-traffic.adoc

```diff
@@ -306,11 +317,40 @@ To configure an external load balancer:
 . Review the information described in *Advanced Options* when adding an external load balancer.
 . Configure DNS before using the CN obtained from the TLS certificate. DNS is required to send requests to applications or API gateways deployed to Runtime Fabric. Add an "A record" to your DNS provider to map the CN to the IP address of the external load balancer or controller VMs.
 
-== Step 5: Deploy Applications
-When you are ready to deploy an application, follow the instructions in xref:deploy-to-runtime-fabric.adoc[Deploy a Mule Application to Runtime Fabric].
+== Step 4: Deploy Applications
+When you are ready to deploy an application:
+
+. Follow the instructions in xref:deploy-to-runtime-fabric.adoc[Deploy a Mule Application to Runtime Fabric].
+. Verify the application URL.
++
+The application's URL contains the routing path for the application. If the default domain is not the desired domain to be served by the application, navigate to the application's *Ingress* tab in Runtime Manager and select the desired domain from the *Domain* drop-down list.
 
 The *Ingress* tab allows you to specify whether inbound traffic should be allowed for the application. After you enable inbound traffic, the default behavior is changed to allow for new application deployments. If there are applications deployed to Runtime Fabric before you enable inbound traffic, they do not receive inbound requests until this setting is enabled.
 
+Use the Ingress tab to update the configuration for application requests.
+
+
+== View TLS Certificates
+
+To view TLS certificate information:
+
+... Select the *Inbound Traffic* tab in Runtime Manager.
```

Comment:
"Select the Inbound Traffic tab in a Runtime Fabric" would be more accurate.

There's no Advanced Options tab. Maybe this should say "Scroll to the domains section"?

Reply:
Updated this - thank you!